test/views/email/CaptureEmailViewSpec.scala

```diff
@@ -116,9 +121,46 @@ class CaptureEmailViewSpec extends ViewBaseSpec {
         }
       }
 
+      "the letterToConfirmedEmail boolean when set to true" should {
+        lazy val view: Html = injectedView(emailForm(testEmail).fill(testEmail),
+          emailNotChangedError = false,
+          currentEmail = testEmail,
+          Call("POST","/vat-through-software/account/correspondence/change-email-address"),
+          letterToConfirmedEmail = true
+        )(user, messages, mockConfig)
+
+        lazy implicit val document: Document = Jsoup.parse(view.body)
+
+        "have a back link" which {
+
+          "directs to the AddEmailAddressController" in {
+            element(Selectors.backLink).attr("href") shouldBe "/vat-through-software/account/correspondence/contact-preference/add-email-address"
+          }
+        }
+      }
+
+      "the letterToConfirmedEmail boolean when set to false" should {
```

Comment:
You're already testing this scenario above, i.e. the back link destination for when the boolean is false (line 72). Looks like this can be removed